README.md

```diff
@@ -22,3 +22,48 @@ ChRIS is comprised of several components and services, and it is designed to hav
   * [pfioh](https://github.com/FNNDSC/pfioh) - Handles input and output of image data into a data center
   * [pman](https://github.com/FNNDSC/pman) - Handles instantiating and checking the status of image processing jobs
   * Image Plugin(s) - An image processor. Turns raw image data into something meaningful for the radiologists/doctors. Ex: [FreeSurfer](https://surfer.nmr.mgh.harvard.edu/)
+
+## Usecase
```

Comment:
Use Case

Reply:
Actually, just remove this since there is only 1

Reply:
@danmcp This is ready to merge